test/CMakeLists.txt

```diff
@@ -32,3 +32,4 @@ function(ADD_HEYOKA_TESTCASE arg1)
 endfunction()
 
 ADD_HEYOKA_TESTCASE(expression)
+ADD_HEYOKA_TESTCASE(evaluate_benchmark_dbl)
```

Comment:
Can you move the code into the benchmark dir instead of the tests? No need to involve catch either, just need a main().